pmd-java/src/test/resources/net/sourceforge/pmd/lang/java/rule/errorprone/xml/AvoidFieldNameMatchingMethodName.xml

```diff
@@ -60,6 +60,48 @@ public class Bar {
         <code><![CDATA[
 public interface Bar {
     public static final int FOO = 5;
+}
+        ]]></code>
+    </test-code>
+
+    <test-code>
+        <description>Test1 in Enum</description>
```

Comment:
```suggestion
        <description>Test1 in Enum #3936</description>
```

Reply:
I will add the message of issue into these three test cases. 😃